composer.json

```diff
@@ -189,7 +189,7 @@
         "spryker/acl-entity-dummy-product": "^0.2.0",
         "spryker/acl-entity-extension": "^1.0.0",
         "spryker/acl-extension": "^1.0.0",
-        "spryker/acl-merchant-portal": "^1.0.0",
+        "spryker/acl-merchant-portal": "dev-master as 1.11.0",
```

Comment:
@asmarovydlo @a-sabaa I had to add this workaround to fix failed tests, because of not released [tests](https://github.com/spryker/acl-merchant-portal/commit/232309edc0fcdce3c84cb26ee45e91a7adbbe13b) from this module.
In suite-nonsplit the problem does not appear.
I expect more releases with the same error because of skipped releases [here](https://github.com/spryker/spryker/pull/10054). Please think about releasing them.

```
Run vendor/bin/console transfer:databuilder:generate
Store: DE | Code bucket: EU | Environment: ci.mysql
[2023-05-22T19:22:19.053930+00:00] Zed.CRITICAL: Exception - Value mismatch for "Merchant.name" transfer property. Value1: "word()"; Value2: "lexify('??????????')". To fix this, search for 'property name="name"' in the code base and fix the wrong one. in "/home/runner/work/suite/suite/vendor/spryker/transfer/src/Spryker/Zed/Transfer/Business/Model/Generator/TransferDefinitionMerger.php::160" {"exception":"[object] (Exception(code: 0): Value mismatch for \"Merchant.name\" transfer property. Value1: \"word()\"; Value2: \"lexify('??????????')\". To fix this, search for 'property name=\"name\"' in the code base and fix the wrong one. at /home/runner/work/suite/suite/vendor/spryker/transfer/src/Spryker/Zed/Transfer/Business/Model/Generator/TransferDefinitionMerger.php:160)
Exception - Exception: Value mismatch for "Merchant.name" transfer property. Value1: "word()"; Value2: "lexify('??????????')". To fix this, search for 'property name="name"' in the code base and fix the wrong one. 
[stacktrace]
```